exam/urls.py

```
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views 
+from django.conf.urls.static import static
```

Ruff (0.8.2)

`django.conf.urls.static.static` imported but unused

Remove unused import: `django.conf.urls.static.static`

(F401)